assets/xml/objects/object_firefly.xml

```diff
@@ -1,54 +1,81 @@
+        <Limb Name="gFireKeeseLeftFootRootLimb" Type="Standard" EnumName="FIRE_KEESE_LIMB_LEFT_FOOT_ROOT" Offset="0x1750" />
+        <Limb Name="gFireKeeseLeftFootWrapperLimb" Type="Standard" EnumName="FIRE_KEESE_LIMB_LEFT_FOOT_WRAPPER" Offset="0x175C" />
+        <Limb Name="gFireKeeseLeftFootLimb" Type="Standard" EnumName="FIRE_KEESE_LIMB_LEFT_FOOT" Offset="0x1768" />
+
+        <!--Body Limb-->
+        <Limb Name="gFireKeeseBodyLimb" Type="Standard" EnumName="FIRE_KEESE_LIMB_BODY" Offset="0x1774" />
+
+        <!--Left Wing Limbs-->
+        <Limb Name="gFireKeeseLeftWingRootLimb" Type="Standard" EnumName="FIRE_KEESE_LIMB_LEFT_WING_ROOT" Offset="0x1780" />
+        <Limb Name="gFireKeeseLeftWingWrapperLimb" Type="Standard" EnumName="FIRE_KEESE_LIMB_LEFT_WING_WRAPPER" Offset="0x178C" />
+        <Limb Name="gFireKeeseLeftWingEndMidRootLimb" Type="Standard" EnumName="FIRE_KEESE_LIMB_LEFT_WING_END_MID_ROOT" Offset="0x1798" />
+        <Limb Name="gFireKeeseLeftWingEndRootLimb" Type="Standard" EnumName="FIRE_KEESE_LIMB_LEFT_WING_END_ROOT" Offset="0x17A4" />
+        <Limb Name="gFireKeeseLeftWingEndLimb" Type="Standard" EnumName="FIRE_KEESE_LIMB_LEFT_WING_END" Offset="0x17B0" />
+        <Limb Name="gFireKeeseLeftWingMidLimb" Type="Standard" EnumName="FIRE_KEESE_LIMB_LEFT_WING_MID" Offset="0x17BC" />
+        <Limb Name="gFireKeeseLeftWingStartLimb" Type="Standard" EnumName="FIRE_KEESE_LIMB_LEFT_WING_START" Offset="0x17C8" />
+
+        <!--Right Wing Limbs-->
+        <Limb Name="gFireKeeseRightWingRootLimb" Type="Standard" EnumName="FIRE_KEESE_LIMB_RIGHT_WING_ROOT" Offset="0x17D4" />
+        <Limb Name="gFireKeeseRightWingWrapperLimb" Type="Standard" EnumName="FIRE_KEESE_LIMB_RIGHT_WING_WRAPPER" Offset="0x17E0" />
+        <Limb Name="gFireKeeseRightWingEndMidRootLimb" Type="Standard" EnumName="FIRE_KEESE_LIMB_RIGHT_WING_END_MID_ROOT" Offset="0x17EC" />
+        <Limb Name="gFireKeeseRightWingEndRootLimb" Type="Standard" EnumName="FIRE_KEESE_LIMB_RIGHT_WING_END_ROOT" Offset="0x17F8" />
+        <Limb Name="gFireKeeseRightWingEndLimb" Type="Standard" EnumName="FIRE_KEESE_LIMB_RIGHT_WING_END" Offset="0x1804" />
+        <Limb Name="gFireKeeseRightWingMidLimb" Type="Standard" EnumName="FIRE_KEESE_LIMB_RIGHT_WING_MID" Offset="0x1810" />
+        <Limb Name="gFireKeeseRightWingStartLimb" Type="Standard" EnumName="FIRE_KEESE_LIMB_RIGHT_WING_START" Offset="0x181C" />
+
+        <!--Head Limbs-->
+        <Limb Name="gFireKeeseHeadRootLimb" Type="Standard" EnumName="FIRE_KEESE_LIMB_HEAD_ROOT" Offset="0x1828" />
+        <Limb Name="gFireKeeseHeadWrapperLimb" Type="Standard" EnumName="FIRE_KEESE_LIMB_HEAD_WRAPPER" Offset="0x1834" />
+        <Limb Name="gFireKeeseHeadLimb" Type="Standard" EnumName="FIRE_KEESE_LIMB_HEAD" Offset="0x1840" />
+
+        <!--Skeleton-->
+        <Skeleton Name="gFireKeeseSkel" Type="Normal" LimbType="Standard" LimbNone="FIRE_KEESE_LIMB_NONE" LimbMax="FIRE_KEESE_LIMB_MAX" EnumName="FireKeeseLimb" Offset="0x18B8" />
```

Comment:
Could you remove `#include "objects/object_firefly/object_firefly.h"` from `z_en_firefly.c` and instead add it to `z_en_firefly.h`? Both `jointTable[FIRE_KEESE_LIMB_MAX]` and `morphTable[FIRE_KEESE_LIMB_MAX]` should have `FIRE_KEESE_LIMB_MAX` instead of `28` in the header

Reply:
should be good now 👍